AWSAuthSDK/AWSAuthSDKTestApp/UserDetailsViewController.swift

```diff
@@ -43,6 +44,16 @@ class UserDetailsViewController: UIViewController {
         updateUserAttributes()
     }
     
+    func listenUserState() {
+        AWSMobileClient.default().addUserStateListener(self) { (state, additionalInfo) in
+
+            if (state == .signedOutUserPoolsTokenInvalid) {
```

Comment:
what if it's another state? could we log the state as an info level log if it is expected to listen to multiple state changes, or log an error level if it's unexpected?

Reply:
This is a hosting app, so doesnot have different log levels. I added a print to log the state so that we can see this in terminal while running the test.